splash/resources.py

```diff
@@ -196,6 +196,13 @@ def _write_expired_args(self, request, expired_args):
         return self._write_error(request, 498, ex)
 
     def _log_stats(self, request, options, error=None):
+
+        if self.hide_passed_json_and_lua_source:
+            if 'posted_json' in options:
+                del options['posted_json']
```

Comment:
I don't think we should be hard-coding specific keys. I can see two ways of handling it:

1) `dont_log` argument, which can disable logging of specific fields per-request (posted_json and lua_source in this case);
2) `--log-args 0` startup argument, which disables logging of args completely

Reply:
Yeah, I didn't think this was necessarily the best way to go about it. I was more so trying to get the idea out and we can go from there.

`--log-args 0` would be great as long as it only stops the logging of this startup/shutdown args and _not_ stops the logging from within the script (e.x. `print()`)

Reply:
I was thinking `--log-args 0` disables logs only for Splash arguments, nothing else - i.e. `"args"` value in the log you've pasted